types/index.d.ts

```diff
@@ -0,0 +1,35 @@
+// TypeScript Version: 3.5
+
+import { Extension, AnySchema } from '@hapi/joi'
```

Comment:
error in my VsCode lint: Parsing error: 'import' and 'export' may appear only with 'sourceType: module'